src/ResourceManager/Sql/Commands.Sql/Instance Failover Group/Cmdlet/RemoveAzureSqlInstanceFailoverGroup.cs

```diff
@@ -0,0 +1,167 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlDatabaseInstanceFailoverGroup",
+        SupportsShouldProcess = true), OutputType(typeof(AzureSqlInstanceFailoverGroupModel))]
```

Comment:
@jugeorge for these cmdlets with multiple parameter sets, please make sure that there is a `DefaultParameterSetName` defined in the `Cmdlet` attribute (which should be set to the parameter set that requires the user to provide a `ResourceGroupName` and `Name`, rather than the `ResourceId` or `InputObject` sets). You can find more information about this [here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#interactive-parameter-set)